bootstrap/pkg/utils/k8utils.go

```diff
@@ -120,24 +121,49 @@ func CreateResourceFromFile(config *rest.Config, filename string, elems ...confi
 	return nil
 }
 
-func GetObjectKindFromUri(configFile string) (string, error) {
+// Checks if the path configFile is remote (e.g. http://github...)
+func IsRemoteFile(configFile string) (bool, error) {
 	if configFile == "" {
-		return "", fmt.Errorf("config file must be a URI or a path")
+		return false, fmt.Errorf("config file must be a URI or a path")
 	}
-
-	appDir, err := ioutil.TempDir("", "")
+	url, err := netUrl.Parse(configFile)
 	if err != nil {
-		return "", fmt.Errorf("Create a temporary directory to copy the file to.")
+		return false, &kfapis.KfError{
+			Code:    int(kfapis.INVALID_ARGUMENT),
+			Message: fmt.Sprintf("Error parsing file path: %v", err),
+		}
+	} else {
```

Comment:
nit: move the code out of the else block since code in the if statement will never reach it.

Reply:
done